package.json

```diff
@@ -4,68 +4,70 @@
-    "@fortawesome/react-fontawesome": "^0.1.4",
-    "@storybook/addon-actions": "5.1.11",
-    "@storybook/addon-info": "5.1.11",
-    "@storybook/addon-knobs": "5.1.11",
-    "@storybook/react": "5.1.11",
-    "@storybook/theming": "5.1.11",
-    "babel-eslint": "10.0.2",
-    "babel-loader": "8.0.6",
-    "eslint": "6.1.0",
-    "eslint-config-airbnb": "^17.1.1",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-loader": "^2.2.1",
-    "eslint-plugin-import": "2.18.2",
-    "eslint-plugin-jsx-a11y": "6.2.3",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "7.14.3",
-    "grunt": "^1.0.4",
+    "@babel/cli": "^7.12.10",
+    "@babel/core": "^7.12.10",
+    "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-react": "^7.12.10",
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@fortawesome/react-fontawesome": "^0.1.14",
+    "@storybook/addon-actions": "5.3.10",
+    "@storybook/addon-info": "5.3.10",
+    "@storybook/addon-knobs": "5.3.10",
+    "@storybook/preset-create-react-app": "^1.5.2",
+    "@storybook/react": "5.3.10",
+    "@storybook/theming": "5.3.10",
+    "babel-eslint": "10.1.0",
+    "babel-loader": "8.1.0",
+    "classnames": "^2.2.6",
+    "eslint": "^7.17.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-prettier": "^7.1.0",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-import": "2.22.1",
+    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "7.22.0",
+    "grunt": "^1.3.0",
     "grunt-contrib-copy": "^1.0.0",
     "grunt-contrib-less": "^2.0.0",
     "grunt-contrib-watch": "^1.1.0",
-    "grunt-stylelint": "^0.11.1",
-    "immutable": "^3.7.3",
-    "mockdate": "^2.0.5",
+    "grunt-stylelint": "^0.16.0",
+    "immutable": "^3.8.2",
+    "mockdate": "^3.0.2",
     "moment": "^2.24.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^1.18.2",
-    "react": "^16.8.6",
+    "prettier": "^2.2.1",
+    "prop-types": "^15.7.2",
+    "react": "~16.8.6",
     "react-bootstrap": "^0.31.5",
-    "react-dom": "^16.8.6",
-    "react-scripts": "^3.1.0",
-    "react-test-renderer": "^16.8.6",
-    "stylelint": "^10.1.0"
+    "react-dom": "~16.8.6",
+    "react-scripts": "^4.0.1",
+    "react-test-renderer": "~16.8.6",
+    "stylelint": "^13.8.0"
   },
   "peerDependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.21",
-    "@fortawesome/free-solid-svg-icons": "^5.10.1",
-    "@fortawesome/react-fontawesome": "^0.1.4",
-    "immutable": "^3.7.3",
-    "moment": "^2.22.2",
-    "react": "^16.8.6",
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@fortawesome/react-fontawesome": "^0.1.11",
+    "classnames": "^2.2.6",
+    "immutable": "^3.8.2",
+    "moment": "^2.24.0",
+    "prop-types": "^15.7.2",
+    "react": "~16.8.6",
```

Comment:
Preco zmena `^` -> `~`, chces ostat na React 16.8?

Reply:
Ty jo, tuším (protože už je to déle) že aby to sedělo s kbc-ui. Kde je 16.8.

Reply:
jj, len teraz to obmedzis na 16.8, takze napr. 16.9 tam nedam. PReferoval by som asi ten predosly zapis. Alebo to prave v novsej nepojde?

Reply:
Určitě to můžeme vrátit, kor když se budeme beztak pokoušet to zvednout. Vrátíš to?
To tam bylo jen fakt asi proto abych měl jistotu že při vývoji tu mám stejnou verzi jakou to bude mít při použití v kbc-ui.

Reply:
Jo, dam to s5

Reply:
No v dev dependencies ju mas, tak ako pises, s `~`. Ale v peer mozeme myslim dovolit aj nieco vyssie.

Reply:
A bol by to BC break, lebo zrazu nepovolujes nic ine okrem 16.8